resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterSchedulerBackend.scala

```diff
@@ -181,7 +184,51 @@ private[spark] class KubernetesClusterSchedulerBackend(
     super.getExecutorIds()
   }
 
+  private def labelLowPriorityExecs(execIds: Seq[String]) = {
```

Comment:
The naming `labelLowPriorityExecs` looks misleading to me because the pods already have `PriorityClass`.

Reply:
Good point. We could say `labelExecsForDecommissioning`?

Reply:
Sounds good to me.